<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="lblYear">
         <property name="text">
          <string>Año</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbYear"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lblBalance">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Balance apuestas del mes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_5">
         <property name="topMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblBet">
           <property name="text">
            <string>Apostado</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtApostado">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblWinnings">
           <property name="text">
            <string>Ganancias</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtGanancias">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblLosses">
           <property name="text">
            <string>Perdidas</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtPerdidas">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblProfits">
           <property name="text">
            <string>Beneficio</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtBeneficio">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblPending">
           <property name="text">
            <string>Pendiente</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="txtPendiente">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblYield">
           <property name="text">
            <string>Yield</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtYield">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblQuota">
           <property name="text">
            <string>Cuota Media</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="txtCuota">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="2" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="topMargin">
          <number>30</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblBets">
           <property name="text">
            <string>Apuestas</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtApuestas">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblWon">
           <property name="text">
            <string>Aciertos</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtAciertos">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblLost">
           <property name="text">
            <string>Fallos</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtFallos">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblNull">
           <property name="text">
            <string>Nulos</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtNulos">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblSuccess">
           <property name="text">
            <string>Acierto</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="txtAcierto">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblAverageBet">
           <property name="text">
            <string>Apuesta Media</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtApuestaMedia">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbMonth"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblMonth">
         <property name="text">
          <string>Mes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbDay"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblDay">
         <property name="text">
          <string>Día</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
